x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -95,4 +177,55 @@ describe('Service Overview', () => {
       });
     });
   });
+
+  describe('Calls APIs', () => {
+    beforeEach(() => {
+      apiRequestsToIntercept.map(({ endpoint, aliasName }) => {
+        cy.intercept('GET', endpoint).as(aliasName);
+      });
+    });
+
+    it('with the correct environment when changing the environment', () => {
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+
+      cy.get('[data-test-subj="environmentFilter"]').select('production');
+
+      cy.expectAPIsToHaveBeenCalledWith({
+        apisIntercepted: aliasNames,
+        value: 'environment=production',
+      });
+    });
+
+    it('when clicking the refresh button', () => {
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+      cy.contains('Refresh').click();
+      cy.wait(aliasNames);
+    });
+
+    it('when selecting a different time range and clicking the update button', () => {
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+
+      cy.selectAbsoluteTimeRange(
+        'Oct 10, 2021 @ 01:00:00.000',
+        'Oct 10, 2021 @ 01:30:00.000'
+      );
+      cy.contains('Update').click();
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+
+      cy.contains('Refresh').click();
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+    });
+
+    it('when selecting a different comparison window', () => {
+      cy.get('[data-test-subj="comparisonSelect"]').should('have.value', 'day');
+
+      // selects another comparison type
+      cy.get('[data-test-subj="comparisonSelect"]').select('week');
+      cy.get('[data-test-subj="comparisonSelect"]').should(
+        'have.value',
+        'week'
+      );
+      cy.wait(aliasNames, { requestTimeout: 10000 });
+    });
```

Comment:
and here:

```
cy.expectAPIsToHaveBeenCalledWith({
        apisIntercepted: aliasNames,
        value: 'comparisonStart=...
      });
```

Reply:
I need to do it just for the endpoint that use the comparison, I'm going to split the request in two lists